### ISSUE TITLE
How can i make Clickable <TR>

### ISSUE THREAD

**Issue by corientdev** [esvit/ng-table#571]
Hi,

Thanks for this wonderful library.
I have one question. How can i make every row clickable??

Regards,
Vishal V.

**Comment by ycheng4** [comment]
I took example 1 in the example site for this repo and did a edit by adding ng-click on the <tr>

http://plnkr.co/edit/ZqB8uCL9Q6byJacmob2f?p=preview

Have a look. This isn't really a issue with ng-table but rather your understanding of angularjs

**Comment by corientdev** [comment]
Yup managed to do it on the same day. Thanks for your time brother.  Appreciate you :-)
